### PR TITLE
Skip migration tests on sig-storage and remove DataVolume custom cleanup

### PR DESCRIPTION
**Pull request by rmohr**
**What this PR does / why we need it**:

Some migration tests were still executed on the sig-storage lane. Since they were executed in parallel there were not enough resources available. Skipping them now.

DataVolumes can take a little bit until they are fully imported.
Increase the timeout, as well as removing the custom cleanup, to collect
in our test reporter the DataVolume, in case something is wrong with the
DataVolume and the test fails.

This helps debugging issues like the one.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/console_test.go [hunk 1]
```diff
@@ -111,9 +104,8 @@ var _ = Describe("[rfe_id:127][posneg:negative][crit:medium][vendor:cnv-qe@redha
 				}
 
 				table.DescribeTable("should return that we are running alpine", func(createVMI vmiBuilder) {
-					vmi, dv := createVMI()
```
Comment (xpivarc) [inline tests/console_test.go:114]: Can we actually modify the `vmiBuilder` please?

**Review by xpivarc — CHANGES_REQUESTED** [review]

File: tests/console_test.go (hunk 1 above)
Comment (rmohr) [reply tests/console_test.go:114]: Done.

**Review by xpivarc — APPROVED** [review]
Thanks!

**Comment by rmohr** [comment]
@xpivarc found the issue: Some migration tests where still executed on the sig-storage lane, and since there are only two nodes the tests were timing out. Can you have another look?

**Review by xpivarc — APPROVED** [review]

**Pull request merged**